tsconfig.json

```diff
@@ -36,8 +36,8 @@
     "allowSyntheticDefaultImports": true,
   },
   "references": [
-    {"path": "./packages/microfronent-sdk"},
-    {"path": "./packages/vite-helpers-console-microfrontend"},
-    {"path": "./packages/webpack-helpers-console-microfrontend"}
+    {"path": "./packages/console-sdk-microfrontend"},
+    {"path": "./packages/console-sdk-vite-helpers"},
+    {"path": "./packages/console-sdk-webpack-helpers"}
```

Comment:
Do not support glob pattern, like: `{"path": "./packages/*"}`?

Reply:
I try to search to typescript docs